db/migrate/20230718173402_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :content
+      t.integer :user_id
```

Comment:
- [ ] Let's rename the comment text field title to text instead of content and prevent future errors as you continue developing your app. 👍🏽